Repository/NpmRepository.php

```diff
@@ -128,9 +129,13 @@ protected function createArrayRepositoryConfig(array $packageConfigs)
         $loader = new ArrayLoader();
 
         foreach ($packageConfigs as $version => $config) {
-            $config['version'] = $version;
-            $config = $this->assetType->getPackageConverter()->convert($config);
-            $packages[] = $loader->load($config);
+            try {
+                $config['version'] = $version;
+                $config = $this->assetType->getPackageConverter()->convert($config);
+                $packages[] = $loader->load($config);
+            } catch (UnexpectedValueException $e) {
+                $this->io->writeError("Composer fails to parse version $version, skipping it");
```

Comment:
error message should contain the name of the repository where the version was found, otherwise such error would be very hard to debug.